### PR TITLE
Improve and clean up documentation.

### PR DESCRIPTION
**Pull request by LyonesGamer**

### CODE REVIEW

File: docs/source/design/models.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+===========
+Data Models
+===========
+
+Below are the database models to be used in TimeSync.
+
+Projects:
+---------
+
+.. code-block:: python
```
Comment (ElijahCaine) [inline docs/source/design/models.rst]: Is there a reason we made these code-blocks instead of tables?

**Comment by ElijahCaine** [comment]
In the future, when we move a file let's not forget about `git mv <filename>`, it makes history easier to keep track of.

**Comment by ElijahCaine** [comment]
lgtm +1

**Comment by ElijahCaine** [comment]
Awesomesauce :+1:

File: docs/source/design/models.rst [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+duration    number of seconds
+user        foreign key to User id
+activity    foreign key to Activity id
+notes       nullable string
+issue_uri   nullable string
+date_worked integer representing millisecond epoch time
+newest      bool
+uuid        UUID string unique to revision set
+revision    integer
+newest      bool
```
Comment (athai) [inline docs/source/design/models.rst]: The newest field is listed twice

**Comment by ElijahCaine** [comment]
We should merge this.

**Pull request merged**